torchvision/transforms/functional.py

```diff
@@ -113,6 +113,11 @@ def pil_to_tensor(pic):
     return img
 
 
+def _is_floating_point(dtype: torch.dtype) -> bool:
+    # helper function since torch.dtype.is_floating_point is not scriptable
+    return isinstance(dtype, (torch.float32, torch.float, torch.float64, torch.double))
```

Comment:
when I was trying this, I found that jit allowed me to use int for dtype, but not torch.dtype.
the dtype itself is an object not a type, so you probably want `in` instead of `isinstance`.
Also we probably want to include half and bfloat16 types

Reply:
As stated in https://github.com/pytorch/vision/pull/2313#issuecomment-643183846 this was pushed prematurely and will be reverted.